src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.ts

```diff
@@ -69,4 +66,7 @@ export class AboutComponent implements OnInit, OnDestroy {
     project.browserOS = browser && browser.os ? browser.os : NOT_AVAILABLE;
     return project;
   }
+  closeModal() {
+    this.close.emit();
+  }
```

Comment:
when we are extending it to the BaseModal, we don't even need this function if I am not wrong. Its already a part of the `BaseModal` class